### PR TITLE
[dialog-node] Add new type definition

### PR DESCRIPTION
**Pull request by nevir**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/dialog-node/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+/**
+ * Information dialog with text and title and an OK button.
+ *
+ * @param msg The message to be displayed in the body of the dialog.
+ * @param title Displayed in the title of the dialog.
+ * @param timeout number of seconds before timing out the dialog. Set to 0 for no timeout.
+ * @param callback called when the dialog closes.
+ */
+export function info(
+	msg: string,
```
Comment (DangerBotOSS) [inline types/dialog-node/index.d.ts:15]: - :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## dialog-node ([<kbd>unpkg</kbd>](https://unpkg.com/browse/dialog-node@latest/))
was missing the following properties:
1. setCwd
1. init
1. debugprint
1. run
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 8c69f93de8c5e7ddf314fbad72addfc065855ca3
</p>

**Comment by peterblazejewicz** [comment]
@nevir can you review callbacks params? Those see to be optional in source code, e.g.:
https://github.com/bat-tomr/dialog-node/blob/master/index.js#L52

**Comment by nevir** [comment]
Good catch! updated to make those optional

**Comment by nevir** [comment]
Ready to merge

**Pull request merged**